### DISCUSSION TITLE
WEBP ImageConversion Doubt

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#5466] [Help]
Im trying to convert the webp image to jpeg and png image format 
but the converted image size(jpeg ,png) is greater than the original image(webp) 

Command using to convert the webp image format to jpeg or png:
convert image.webp image1.jpeg
convert image.webp image2.png

size of jpeg and png image is greater than the webp image 

How to reduce the image size while converting from webp to jpeg or png ?

**Comment by fmw42** [comment]
What version of Imagemagick and what platform/OS and what version of the webp delegate library are you using. Is webp up to date current.

**Reply by programmerarunkumar** [reply]
ImageMagick  version = 7.1.0-47
Linux

**Reply by fmw42** [reply]
Post a webp image that has this issue

**Reply by programmerarunkumar** [reply]
will share

**Reply by programmerarunkumar** [reply]
> Post a webp image that has this issue

https://drive.google.com/file/d/1ouNraZAd3HPawoOb0-S5-ma0mEHkgpJZ/view?usp=sharing
when converting this webp image to png(size of png is around 19MB)
when converting this webp image to jpeg(size of jpeg is around 4MB)

**Reply by fmw42** [reply]
JPEG is lossy compressed and can compress to a smaller file size. PNG is non-lossy compressed and does not compress the file size that much.

**Reply by programmerarunkumar** [reply]
so while converting the webp image to jpeg or png image the size of jpeg or png image will  be larger than webp image ?

**Reply by fmw42** [reply]
WEBP was introduced as a better compression method than PNG or JPG.  (JPG was compressed at default 92 quality. You can make it smaller than webp by lowering the quality)

**Comment by snibgo** [comment]
PNG compression is designed for graphics (such as large areas of constant colours), not photos.

JPG compression is designed for photos. A suitable "-quality XX" setting will reduce the file size below the size of the webp.

**Comment by programmerarunkumar** [comment]
What is the  best quality range we can use during the image conversion from webp to jpeg so that the quality of an image will not reduced ?

**Comment by snibgo** [comment]
IM always writes JPG files with lossy compression. To minimise the loss, use "-quality 100".

**Reply by programmerarunkumar** [reply]
IM will use default JPG quality 92 while converting the image from webp to jpg ?

**Reply by fmw42** [reply]
I suspect so. But why do you not just try it. That is faster than waiting for answers.
